3684/0/about_dice_project.rb

```diff
@@ -0,0 +1,81 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+# :reek:FeatureEnvy
+# :reek:TooManyStatements
+# :reek:UncommunicativeMethodName
+# :reek:InstanceVariableAssumption
+
+class DiceSet
+  attr_reader :values
+  def roll(value)
+    @values = []
+    iterator = 0
+    while iterator < value
+      @values << rand(1..6)
+      iterator += 1
+    end
+  end
```

Comment:
please take a look at http://ruby-doc.org/core-2.6.3/Array.html#method-c-new

as you can see there, a structure like `Array.new(num_of_rolls) { rand(1..6) }` looks much better here) but it's up to you)
just give me your opinion. I'm gonna approve anyway)

Reply:
Yep, this is much better. Thanks for helping